### DISCUSSION TITLE
Problems after installation

### DISCUSSION THREAD

**Discussion by timster00** [Homebrew/discussions#1244] [Getting started]
Hi everyone!

I have installed HomeBrew on M1 Mac for the first time, i dont know how to code. I just want to check the status of SSD. After installation there is a message "installation successful" but i cant use any command. I wrote "brew help" and I got "zsh: command not found: brew". What should I do? I took the installation script from the official site

**Comment by carlocab** [comment]
Did you follow the instructions to add `brew` to your `PATH` shown at the end of the install? If you did, restart your terminal. If you didn't, do
```
echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> ~/.zprofile
```
and then restart your terminal.

See also https://github.com/Homebrew/discussions/discussions/775.

**Reply by timster00** [reply]
Thanks a lot!
